src/pdn/doc/add_pdn_connect.md

```diff
@@ -0,0 +1,46 @@
+## add_pdn_connect
+
+### Synopsis
+```
+  % add_pdn_connect
+    [-grid grid_name] \
+    [-layers list_of_two_layers] \
+    [-cut_pitch pitch_value] \
+    [-fixed_vias list_of_fixed_vias]
+```
+
+### Description
+
+The add_pdn_connect command is used to define which layers in the power grid are to be connected together. During power grid generation, vias will be added for overlapping power nets and overlapping ground nets. The use of fixed vias from the technology file can be specified or else via stacks will be constructed using VIARULEs. If VIARULEs are not available in the technology, then fixed vias must be used.
```

Comment:
I would suggest wrapping commands and arguments in paragraphs with a single backtick (`) so we have a nicer presentation on github and readthedocs.

Reply:
Nice suggestion